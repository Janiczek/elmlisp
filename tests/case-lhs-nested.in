(case maybe
  (Nothing 1)
  ((Just val) val))
