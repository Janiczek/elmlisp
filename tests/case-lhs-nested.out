case maybe of
    Nothing ->
        1

    Just val ->
        val
